### ISSUE TITLE
Enable text-wrapping

### ISSUE THREAD

**Issue by strout18** [penrose/penrose#349]
I think it would be useful to auto-text-wrap once a label reaches a certain size (either user-specified or hardcoded). Perhaps the ability to disable it would also be useful.

**Comment by maxkrieger** [comment]
ref: http://docs.mathjax.org/en/latest/output/linebreaks.html

**Comment by joshsunshine** [comment]
This is very hard for now due to MathJax problems. Closing for now (#653).
